src/adapters/openai/helpers/embedding.ts

```diff
@@ -0,0 +1,25 @@
+import OpenAI from "openai";
+import { Context } from "../../../types";
+import { SuperOpenAi } from "./openai";
+const VECTOR_SIZE = 3072;
```

Comment:
We could consider having this parameter in the configuration later for easy changes and testing.

Reply:
I could add this in env for now. Is there a global config file ?